app/models/mets_document/mods_document.rb

```diff
@@ -29,6 +29,21 @@ def title
           title
         end
       end
+      non_sort = value_from xpath: "mods:titleInfo/mods:nonSort"
+      return "#{non_sort} #{title_value}".gsub(/["\[\]]/, "") unless non_sort.empty?
+      title_value
+    end
+
+    def publisher
+      values = value_from xpath: "mods:originInfo/mods:publisher"
+      values.join(" - ")
+    end
+
+    def published_date
+      date_array = normalize_whitespace(value_from(xpath: "mods:originInfo/mods:dateOther") +
```

Comment:
I think these should be separate fields, rather than all combined into `published_date`

Reply:
I will add them as separate fields and display them separately. correct?

Reply:
:+1: